helm/flink/templates/deployment-jobmanager.yaml

```diff
@@ -10,6 +10,16 @@ spec:
         app: {{ template "fullname" . }}
         component: jobmanager
     spec:
+      initContainers:
+      - name: flink-copy-libs
+        image: {{ .Values.image }}:{{ .Values.imageTag }}
+        command:
+        - sh
+        - -c
+        - cp /opt/flink/lib/* {{ range .Values.flink.additional_libs}}/opt/flink/opt/{{ . }} {{ end }} /flink-lib;
```

Comment:
minor style: space after `additional_libs` (also in `deployment-taskmanager.yaml`)